Python/codegen.c

```diff
@@ -4141,39 +4141,38 @@ codegen_sync_comprehension_generator(compiler *c, location loc,
     comprehension_ty gen = (comprehension_ty)asdl_seq_GET(generators,
                                                           gen_index);
 
-    if (!iter_on_stack) {
-        if (gen_index == 0) {
+    if (gen_index == 0) {
+        if (!is_inlined) {
```

Comment:
Why flip these two cases? Aren't they equivalent?

Reply:
Two cases were flipped for impoving readability.
They were equivalent, but with special trick for this, value of iter_on_stack was forced to be 0 if gen_index > 0.
And this is not very clear, IMO.
For now value of is_inlined is not changed from the very high level and those aren't equivalent.